.github/wiki/FirstPullRequest.md

```diff
@@ -0,0 +1,120 @@
+
+You can now make any changes deemed necessary using your code editor of choice. Many popular editors integrate with source control in some way. You can also use `git status` and `git diff` on the command line to keep track of what has changed.
+
+### 2. Test your changes
+
+Make sure your changes are correct and do not introduce any test failures. You can learn more in [Running and Writing Tests](https://github.com/facebook/react-native/wiki/Tests).
+
+### 3. Lint your code
+
+We understand it can take a while to ramp up and get a sense of the style followed for each of the languages in use in the core React Native repository. Developers should not need to worry about minor nits, so whenever possible, we use tools that automate the process of rewriting your code to follow conventions.
+
+For example, we use [Prettier](https://prettier.io/) to format our JavaScript code. This saves you time and energy as you can let Prettier fix up any formatting issues automatically through its editor integrations, or by manually running `npm run prettier`. We also use a linter to catch styling issues that may exist in your code. You can check the status of your code styling by simply running `npm run lint`.
+
+To learn more about coding conventions, refer to the [Style Guide](#style-guide).
+
+## Chapter III: Proposing your Changes
+
+### 1. Commit your changes
+
+Make sure to add your changes to version control using `git`:
+
+```bash
+git add <filename>
+git commit -m <message>
+```
+
+You can use a short descriptive sentence as your commit message.
+
+> Worried about writing good git commit messages? Do not fret. Later, when your pull request is merged, all your commits will be squashed into a single commit. It is your pull request description which will be used to populate the message for this squashed commit.
+
+This guide covers just enough to help you along with your first contribution. GitHub has several resources to help you get started with git:
+
+- [Using Git](https://help.github.com/en/categories/using-git)
+- [The GitHub Flow](https://guides.github.com/introduction/flow/)
+
+
+### 2. Push your changes to GitHub
+
+Once your changes have been commited to version control, you can push them to GitHub.
+
+```bash
+git push fork <my_feature_branch>
+```
+
+If all goes well, you will see a message encouraging you to open a pull request:
+
+```bash
+remote:
+remote: Create a pull request for 'your_feature_branch' on GitHub by visiting:
+remote:      https://github.com/your_username/react-native/pull/new/your_feature_branch
+remote:
+```
+
+Visit the provided URL to proceed to the next step.
+
+### 3. Create your pull request
+
+You are almost there! The next step is to fill out the pull request. Use a descriptive title that is not too long. Then, make sure to fill out all of the fields provided by the default pull request template:
+
+**Summary:** Use this field to provide your motivation for sending this pull request. What are you fixing?
```

Comment:
Just noting that by putting the template inline it will need to be maintained when we change the template, may consider linking to a new PR